tools/train.py

```diff
@@ -169,6 +174,7 @@ def main():
 
     # set random seeds
     seed = init_random_seed(args.seed)
```

Comment:
Can this be replaced with sync_random_seed?

Reply:
May `init_random_seed` not be equal to`sync_random_seed`?

Reply:
The logic is not the same, and bc breaking is not expected.